dapps/marketplace/src/pages/listings/ListingCards.js

```diff
@@ -21,59 +24,75 @@ const ListingCards = ({
   const [redirect, setRedirect] = useState()
   if (!listings) return null
 
+  const listingToRender = [
```

Comment:
Can we hide these on desktop for now?

Reply:
I guess @micahalcorn wanted them to be on Desktop also.

@micahalcorn Any thoughts on this?

Reply:
@nick @micahalcorn Have disabled this everywhere other than home page on Mobile for now. Let me know if this has to be enabled in other places.